src/App.jsx

```diff
@@ -1,3 +1,38 @@
-export default function App() {
-  return <h1 className="text-3xl font-bold underline bg-blue-500">React exercises</h1>;
-}
+import CardComponent  from "./components/CardComponent ";
+import data from "./data";
+
+const App = () => {
+  const genLinks = () => {
+    let arr = [];
+    for (let i = 0; i < 10; i++) {
+      arr.push({ text: `Exercise ${i}`, link: `/exercise${i}` });
+    }
+    return arr;
+  };
+
+  const array = genLinks();
```

Comment:
You should use the descriptive names for te variables such as genLinks, arr, array and i